data/data_fixes/fix_sample_ids.rb

```diff
@@ -0,0 +1,34 @@
+require 'csv'
+require 'logger'
+
+ActiveRecord::Base.logger = nil
+logger = Logger.new("sample_ids_log.log")
+data = CSV.parse(File.read("data/data_fixes/root_sample_ids.csv"), headers: true)
+failed_ids = Array.new()
+success_ids = Array.new()
+
+data.each_with_index do |row, index|
+    if index % 1000 == 0
+        puts "."
+    end
+    samples = Sample::Metadata.where("sample_description" => row["original_root_sample_id"])
```

Comment:
It will be interesting to test this on training - I think it will be v slow, because sample_description isn't indexed (and can't be I think because it's a long text field), but I can't think of a better option 😆

Reply:
I just tested this approach with a single record on UAT, and it updated the 'updated_at' timestamp, and updated the MLWH 👍